package.json

```diff
@@ -13,7 +13,9 @@
     "watch": "tsc -w",
     "dev": "npm run build && concurrently -k -p \"[{name}]\" -n \"TypeScript,App\" -c \"yellow.bold,cyan.bold\" \"npm:watch\" \"npm:dev:start\"",
     "dev:start": "npm run build && fastify start --ignore-watch=.ts$ -w -l info -P dist/app.js",
-    "test": "npm run db:seed && tap --jobs=1 test/**/*",
+    "test": "npm run db:seed && npm run test:run",
+    "test:coverage": "npm run db:seed && c8 npm run test:run",
+    "test:run": "glob -c \"tsx --test\" \"./test/**/*.integration.spec.ts\"",
```

Comment:
What is this `.integration.spec`?
Did you run the tests locally?

Reply:
My IDE automatically added this during refactoring. Sorry for letting this pass. It's surprising that all tests ran and passed despite this issue.